unit_tests/test_tlslite_extensions.py

```diff
@@ -1679,5 +1680,53 @@ def test_parse_with_trailing_data(self):
         with self.assertRaises(SyntaxError):
             self.ext.parse(parser)
 
+class TestHeartbeatExtension(unittest.TestCase):
+    def test___init___(self):
+        ext = HeartbeatExtension()
+
+        self.assertIsNotNone(ext)
+        self.assertEqual(ext.extType, 15)
+        self.assertIsNone(ext.mode)
+
+    def test_create(self):
+        ext = HeartbeatExtension().create(0)
```

Comment:
this should use the constant so that we're not only testing that the values are passed through, but that the constants are correct